### PR TITLE
Bump OCP version to 4.7

### PR DESCRIPTION
**Pull request by vimalk78**
Bumped OCP version to 4.7
/cc @igor-karpukhin 
/assign @jcantrill

### CODE REVIEW

**Review by jcantrill — COMMENTED** [review]
/hold

File: manifests/4.7/cluster-logging.v4.7.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -159,9 +159,9 @@ spec:
   - type: SingleNamespace
     supported: true
   - type: MultiNamespace
-    supported: true
+    supported: false
   - type: AllNamespaces
-    supported: true
+    supported: false
```
Comment (igor-karpukhin) [inline manifests/4.7/cluster-logging.v4.7.0.clusterserviceversion.yaml:164]: @vimalk78  Are we dropping support of `MultipleNamespace` as well as `AllNamespaces`?

Comment (periklis) [reply manifests/4.7/cluster-logging.v4.7.0.clusterserviceversion.yaml:164]: We actually never supported this one, right? cluster-logging-operator was always capable to handle requests in `openshift-logging`.

**Comment by jcantrill** [comment]
Closing in favor of #755

**Pull request closed without merging**